activerecord/lib/active_record/tasks/database_tasks.rb

```diff
@@ -309,11 +309,12 @@ def class_for_adapter(adapter)
         def each_current_configuration(environment)
           environments = [environment]
           environments << "test" if environment == "development"
+          environments.each do |env|
```

Comment:
Had to manually make this change due to https://app.circleci.com/pipelines/github/procore/procore/369101/workflows/f64704bf-b6c4-4f3f-8c59-0e1566cf2f24/jobs/5068344

When I tried checking out this file from 6-0-stable, there were a lot of new features that expanded the scope greatly